packages/taquito-rpc/src/types.ts

```diff
@@ -11,6 +11,80 @@ export type DelegateResponse = string | null;
 
 export type OperationHash = string;
 
+interface Before {
+  value?: string;
+  node?: string;
+}
+
+interface After {
+  value?: string;
+  node?: string;
+}
+
+interface INodeExtender {
+  length: number;
+  segment: string;
+  proof: string;
+}
+
+interface OtherEltsInner {
```

Comment:
OtherEltsInner will be `value` or `inode_extender`
It could be represented as follow:
```
type OtherEltsInner = {
                value: any
            } | {
                inode_extender?: INodeExtender;
            }
```